src/modules/logbook/LogBook.ts

```diff
@@ -17,9 +18,11 @@ export default class LogBook implements Feature {
     public filteredLogs: PureComputed<LogBookLog[]> = ko.pureComputed(() => this.logs().filter((log) => this.filters[log.type.label]?.()));
 
     newLog(type: LogBookType, content: LogContent) {
-        const length = this.logs.unshift(new LogBookLog(type, content));
-        if (length > 1000) {
-            this.logs.pop();
+        if (this.canLog(`logBook.${type.label}`)) {
```

Comment:
I don't like how you have to know how the settings names are set up to call `canLog` - I would prefer if `canLog` had the whole LogBookType as the parameter and then that function can do the dirty work of calculating the setting name we need.